app/js/instances/instanceDetailController.js

```diff
@@ -83,6 +81,14 @@ function instanceDetailController($scope, $routeParams, $timeout, flux, dockerSe
             }
         });
     };
+
+    $scope.removeImage = function(image) {
+        image.loading = true;
+        dockerService.d('images.remove', {
+            host: $routeParams.host,
```

Comment:
@ngmiller I wasn't able to use $scope.instance.alias here for some reason. It was `undefined` for some reason.